common/src/main/java/org/cloudfoundry/autosleep/access/cloudfoundry/CloudFoundryApi.java

```diff
@@ -54,26 +36,38 @@
 import org.cloudfoundry.client.v2.events.ListEventsResponse;
 import org.cloudfoundry.client.v2.organizations.GetOrganizationRequest;
 import org.cloudfoundry.client.v2.organizations.GetOrganizationResponse;
-import org.cloudfoundry.client.v2.routes.GetRouteRequest;
-import org.cloudfoundry.client.v2.routes.GetRouteResponse;
-import org.cloudfoundry.client.v2.routes.ListRouteApplicationsRequest;
-import org.cloudfoundry.client.v2.routes.ListRouteApplicationsResponse;
-import org.cloudfoundry.client.v2.routes.RouteEntity;
+import org.cloudfoundry.client.v2.privatedomains.GetPrivateDomainRequest;
+import org.cloudfoundry.client.v2.privatedomains.GetPrivateDomainResponse;
```

Comment:
Looks like an unused import since we are still using the deprecated GetDomains API.